src/components/MarketWidget/MarketsWidget.tsx

```
@@ -0,0 +1,2309 @@
+    const baseAsset = item.baseAsset;
+    const quoteAsset = item.quoteAsset;
+    
+    // Check if query matches ticker symbols (case optimized)
+    if (baseAsset.toLowerCase().includes(normalizedQuery)) return true;
+    if (quoteAsset.toLowerCase().includes(normalizedQuery)) return true;
+    
+    // Check if query matches full asset names - only do this lookup if necessary
+    const baseAssetFullName = (baseAsset in ASSETS) ? 
+      ASSETS[baseAsset]?.name.toLowerCase() || '' : '';
+    
+    if (baseAssetFullName && baseAssetFullName.includes(normalizedQuery)) return true;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)